packages/web-ui/src/lab/TextLink/TextLink.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { ElementRef, forwardRef, PropsWithChildren } from 'react';
+import { PropsWithSx } from '../../types';
+import { getPrefixedName, px } from '../../utils';
+import clsx from 'clsx';
+import { Typography } from '../../Typography';
+import { TextLinkProps } from './TextLink.props';
+import { styled } from '../../theme';
+import { colors, colorsCommon } from '@utilitywarehouse/colour-system';
+import { useBackground } from '../../Box';
+
+const componentName = 'TextLink';
+const componentClassName = getPrefixedName(componentName);
+
+const StyledTypography = styled(Typography)(() => {
+  const { isBrandBackground } = useBackground();
+  return {
+    '--text-link-color-default': isBrandBackground ? colorsCommon.brandWhite : colors.cyan600,
```

Comment:
So maybe Im looking it at the wrong thing 🤔  but I don't see the same `:visited` styles as in Figma `cyan.800` and `white`. Maybe I'm looking at an old build?